packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/catalog/services.py

```diff
@@ -52,6 +53,7 @@ async def _call(
             user_id=user_id,
             limit=limit,
             offset=offset,
+            timeout_s=2 * RPC_REQUEST_DEFAULT_TIMEOUT_S,
```

Comment:
Magic number multipllier `2` hidden in code detected ⚠️

Reply:
thx. this is under control :-)